test/util/shorthand_parser_util.js

```diff
@@ -47,6 +47,7 @@ describe("ShorthandParserUtil", function () {
                 branches: {},
                 remotes: {},
                 index: {},
+                notes:{},
```

Comment:
eslint has an option to enforce this; jshint apparently doesn't. So that's one argument for switching.